packages/cli/src/constructs/api-check.ts

```diff
@@ -193,7 +193,11 @@ export interface Request {
   url: string,
   method: HttpRequestMethod,
   followRedirects?: boolean,
+  /**
+   * To be deprecated, use `skipSSL`
+   */
   skipSsl?: boolean,
```

Comment:
Since we directly have an error when we set this value atm, we can just delete it already without deprecation

Reply:
@umutuzgur , now, the `skipSsl` is deprecated in this PR, if someone is using the `skipSsl` will get:

![image](https://user-images.githubusercontent.com/5315705/226619782-cce8ffa1-1ad8-48ee-b2d8-14ba5e9e4e1a.png)

Is this OK?

Reply:
Let me triple check with Tim

Reply:
Wasn't also `skipSsl: false` throwing the error Tim reported?

Reply:
@umutuzgur , there were two related issues:
1. if the SSL is wrong, the response format crashed because there is no `response.body`, fixed in `reporters/util.ts` file
2. the `skipSsl` wasn't working as expected (wrong property case), fixed be renaming the prop

If the `skipSsl` would have been working, the `skipSsl: true` would avoid returning an error. Using `skipSsl: false` would have crashing the CLI too.